hazelcast/src/test/java/com/hazelcast/internal/serialization/PortableClassVersionTest.java

```diff
@@ -157,10 +158,10 @@ static void testDifferentClassVersionsUsingDataWriteAndRead(SerializationService
 
     @Test
     public void testPreDefinedDifferentVersionsWithInnerPortable() {
-        final SerializationService serializationService = createSerializationService(1);
+        final SerializationService serializationService = PortableTest.createSerializationService(1);
```

Comment:
The static import is there but unused (4 times), also lost import of `createNamedPortableClassDefinition`.